djangocms_fil_permissions/permissions.py

```diff
@@ -0,0 +1,40 @@
+from django.core.exceptions import PermissionDenied
+
+from rules.rulesets import RuleSet
+
+from .rules import has_site_access
+
+
+site_permissions = RuleSet()
+site_permissions.add_rule("site_perm", has_site_access)
+
+
+class SitePermissionBackend(object):
```

Comment:
Please explain how this backend links into the app / is to be used. If it's documented elsewhere, please link to those docs in the docstring.

Reply:
https://docs.djangoproject.com/en/2.1/ref/contrib/auth/#module-django.contrib.auth.backends

Reply:
This is already covered in README and docs/index